src/test/test-execute.c

```diff
@@ -209,7 +258,7 @@ int main(int argc, char *argv[]) {
         }
 
         assert_se(setenv("XDG_RUNTIME_DIR", "/tmp/", 1) == 0);
-        assert_se(set_unit_path(TEST_DIR) >= 0);
+        assert_se(set_unit_path(TEST_DIR "/test-execute/") >= 0);
```

Comment:
@ronnychevalier , `set_unit_path` doesn't overwrite `SYSTEMD_UNIT_PATH` and `test-execute` fails (see #1734). Please, take a look. What do you think?